build/build.go

```diff
@@ -336,7 +336,7 @@ func toSolveOpt(ctx context.Context, d driver.Driver, multiDriver bool, opt Opti
 		}
 	}()
 
-	if opt.ImageIDFile != "" {
+	if len(opt.ImageIDFile) > 0 {
```

Comment:
hmm.., why this change(s)? The previous one is much clearer imho, and type safe.

Reply:
Reverted. Was using the same check for the previous impl for metadata-file, forgot to cleanup. It's more a matter of personal taste but Go compiler generates [identical code](https://godbolt.org/z/fib1x1).